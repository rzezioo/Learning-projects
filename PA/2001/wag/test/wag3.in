3
100 10
2 5
1 4
