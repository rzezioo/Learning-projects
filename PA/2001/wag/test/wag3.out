1 4
2 5
100 10
